machine.mk

```diff
@@ -38,7 +38,7 @@ endif
 
 # To switch machines, simply switch the path of BSG_MACHINE_PATH to
 # another directory with a Makefile.machine.include file.
-BSG_MACHINE_PATH ?= $(BSG_F1_DIR)/machines/pod_X1Y1_ruche_X16Y8_hbm
+BSG_MACHINE_PATH ?= $(BSG_F1_DIR)/machines/bigblade_pod_X1Y1_ruche_X16Y8_hbm
```

Comment:
Is this intentional? I thought if it was, it would go in #744

Reply:
removed. not intentional.